### PR TITLE
New form

### PR DESCRIPTION
**Pull request by marouanebr**

### CODE REVIEW

File: form.component.spec.ts [hunk 1]
```diff
@@ -8,13 +8,14 @@ describe('FormComponent', () => {
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      declarations: [ FormComponent ]
+      declarationsssss: [ FormComponent ]
```
Comment (Mayssabr) [inline form.component.spec.ts:11]: erreur "ssss"

File: form.component.spec.ts [hunk 2]
```diff
@@ -8,13 +8,14 @@ describe('FormComponent', () => {
...
     })
     .compileComponents();
   });
 
   beforeEach(() => {
     fixture = TestBed.createComponent(FormComponent);
+	print
```
Comment (Mayssabr) [inline form.component.spec.ts:18]: not exisit

**Review by Mayssabr — CHANGES_REQUESTED** [review]

File: form.component.spec.ts (hunk 1 above)
Comment (marouanebr) [reply form.component.spec.ts]: ok sala7tha

File: form.component.spec.ts (hunk 2 above)
Comment (marouanebr) [reply form.component.spec.ts]: je pense elli nest7a9ha mais ok fasa5tha

File: form.component.html [hunk 3]
```diff
@@ -1 +1,3 @@
 <p>form works!</p>
+<ha>form works!<ha>
```
Comment (Mayssabr) [inline form.component.html:2]:
```suggestion
<ha>form works!</ha>
```

**Review by Mayssabr — CHANGES_REQUESTED** [review]

File: form.component.html (hunk 3 above)
Comment (marouanebr) [reply form.component.html:2]: no

**Pull request merged**